bin/deploy

```diff
@@ -59,4 +59,15 @@ class Deploy < Thor
   end
 end
 
+def warning
+  puts '*************************'
+  puts 'WARNING! (added jan 2022)'
+  puts "Argo is commented out in config/settings.yml since we don't want to"
+  puts 'automatically deploy Argo during the Fedora removal workcycle!'
```

Comment:
deploy Argo to production

Reply:
Yeah, a note about still wanting to manually deploy to QA and stage would be good to see.